.gitignore

```diff
@@ -1,2 +1,3 @@
 /vendor
 composer.lock
+.*.sw?
```

Comment:
This is probably best handled in your global git configuration. It's been considered "best practice" to leave out environment-specific ignores from the project's git-ignore.